nix/haskell.nix

```diff
@@ -56,8 +56,8 @@ let
     src = ../.;
     # our current release compiler is 8107
     compiler-nix-name = lib.mkDefault "ghc8107";
-    # but we also build for 962.
-    flake.variants = lib.genAttrs ["ghc962"] (x: {compiler-nix-name = x;});
+    # but we also build for 927.
+    flake.variants = lib.genAttrs ["ghc927"] (x: {compiler-nix-name = x;});
```

Comment:
Can't we have `["ghc927" "ghc962"]` here? I'm a bit concerned that we'd regress on all the changes @erikd already made?

Reply:
We're stuck in an awkward place on this one for the time being.  Postgres won't build with musl since this commit: https://github.com/NixOS/nixpkgs/commit/8d4a81b1851be7823e23d26bd02abcfb902d2bef, or at least any version < 14, which means we can't update nixpkgs-unstable. The revision of nixpkgs we're using does not contain ghc962 (and I'm having trouble getting ghc961 to build).

I've submitted an issue and PR to nixpkgs, hopefully someone will look at it. I'm tracking everything here: https://github.com/input-output-hk/cardano-db-sync/issues/1483

Reply:
@angerman nixpkgs-unstable should be fixed now, how often does haskell.nix update?

Reply:
@sgillespie whenever @hamishmack bumps it 🙈